logicmonitor/provider.go

```diff
@@ -67,9 +69,9 @@ func Provider() *schema.Provider {
 func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
-	id := d.Get("api_id").(string)
-	key := d.Get("api_key").(string)
-	company := d.Get("company").(string) + ".logicmonitor.com"
+ 	id := d.Get("api_id").(string)
```

Comment:
Please remove all the whitespace changes from this file.

Reply:
Updated thee pull request.